src/var/intervalSet.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
```

Comment:
One day we should really decide if we're going to use those new spacing rules or not: `define( function() {`. We're inconsistent at the moment. It's the latest style guide but nobody on the team seems to like it.